Automattic/jetpack
# Dashboard: Update Apps Card

#### Changes proposed in this Pull Request:

* Move the AppsBadge component to the general components folder
* Adds the option to pass down an `onBadgeClick` prop to `AppsBadge`
* Adds a README.md file for `AppsBadge`
* Updates the AppsCard component copy and adds buttons to the Android and iOS app stores

#### Jetpack product discussion


Context: 1198218726984184-as-1201477044754576/f

#### Does this pull request change what data or activity we track or use?


It adds a `store` property to `jetpack_wpa_click` (currently tracked by `target = 'apps-card'`), and fixes the `page` property.

#### Testing instructions:






|Desktop|Mobile|
|------|------|
|![image](https://user-images.githubusercontent.com/5714212/145630732-4128d646-627d-439d-8d48-59ed5d3949ba.png) | ![image](https://user-images.githubusercontent.com/5714212/145630885-7d52a649-df4d-4a4d-924b-bffab3d09856.png)|

* Go to Jetpack Dashboard: `/wp-admin/admin.php?page=jetpack#/dashboard`
* Scroll to the bottom of the page where you should see the Apps card
* Make sure the copy and design matches the screenshot above, the store badges link to the correct store, and are tracked correctly

Note: The changes done here are compatible with the code in `projects/plugins/jetpack/_inc/client/recommendations/sidebar/mobile-app/index.js` without major changes. If you want to test `AppsBadge` manually in that component, the easiest way of doing that is by hardcoding `sidebarCard = <MobileApp />` [here](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/_inc/client/recommendations/summary/index.jsx#L131) and  going to `/wp-admin/admin.php?page=jetpack#/recommendations/summary` to verify that it looks and works as expected.
